cmd/vic-machine-server/Dockerfile

```diff
@@ -7,11 +7,17 @@
 
 FROM photon:2.0
 
+RUN mv /etc/yum.repos.d /etc/yum.repos.d.bak
+RUN mkdir -p /etc/yum.repos.d
+COPY isos/base/repos/photon-2.0/photon-2.0.repo.local /etc/yum.repos.d/
+RUN mv /etc/yum.repos.d/photon-2.0.repo.local /etc/yum.repos.d/photon-2.0.repo
 RUN set -eux; \
      tdnf distro-sync --refresh -y; \
      tdnf install shadow -y; \
      tdnf info installed; \
      tdnf clean all
+RUN rm -rf /etc/yum.repos.d
+RUN mv /etc/yum.repos.d.bak /etc/yum.repos.d
```

Comment:
Why we need to move back?

Reply:
photon-2.0.repo.local is for local yum repo, we should restore repo to official repo after packages are installed.